shell/initialize/client.js

```diff
@@ -35,12 +34,6 @@ if (!global.fetch) {
 let app;
 let router;
 
-const $config = nuxt.publicRuntimeConfig || {}; // eslint-disable-line no-undef
-
-if ($config._app) {
-  __webpack_public_path__ = urlJoin($config._app.cdnURL, $config._app.assetsPath); // eslint-disable-line camelcase, no-undef
```

Comment:
Neither `cdnUrl` not `assetsPath` is defined anywhere in our app making this code useless.